src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1684,12 +1696,23 @@ because xbuild doesn't support framework reference assemblies.
     _GenerateAndroidRemapNativeCode;
     _GenerateEmptyAndroidRemapNativeCode;
     _IncludeNativeSystemLibraries;
+    _GetGeneratePackageManagerJavaInputs;
   </_GeneratePackageManagerJavaDependsOn>
 </PropertyGroup>
 
+<Target Name="_GetGeneratePackageManagerJavaInputs">
+  <ItemGroup>
+    <_GeneratePackageManagerJavaInputs Include="@(_AndroidMSBuildAllProjects)" />
+    <_GeneratePackageManagerJavaInputs Include="$(_ResolvedUserAssembliesHashFile)" />
+    <_GeneratePackageManagerJavaInputs Include="@(_ResolvedUserMonoAndroidAssemblies)" />
+    <_GeneratePackageManagerJavaInputs Include="$(_AndroidBuildPropertiesCache)" />
+    <_GeneratePackageManagerJavaInputs Include="@(AndroidEnvironment);@(LibraryEnvironments)" Condition=" '$(EmbedAssembliesIntoApk)' == 'true' " />
+  </ItemGroup>
+</Target>
```

Comment:
Can both targets use the same item group?

I'm thinking about:
* https://github.com/dotnet/android/pull/9174

Where there was a bug if `_GenerateJavaStubs` runs and `_GeneratePackageManagerJava` doesn't.

Reply:
Or maybe that was vice versa.

Reply:
There is still the Manifest which is different between the two. But maybe one could "use" the other and add the manifest.